### PR TITLE
(FIX): put padding so the text is not clipped

### PR DESCRIPTION
**Pull request by dobri1408**
BEFORE: 
Captură de ecran din 2023-10-11 la 10 14 36

AFTER:
Captură de ecran din 2023-10-11 la 10 13 20

### CODE REVIEW

File: src/components/manage/Blocks/Search/components/SelectStyling.jsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const customSelectStyles = {
   valueContainer: (styles) => ({
     ...styles,
     padding: '0px',
+    paddingTop: '4px',
     paddingLeft: 0,
```
Comment (ichim-david) [inline src/components/manage/Blocks/Search/components/SelectStyling.jsx:36]: @dobri1408  this is redundant given that at line 34 you have padding 0

Comment (ichim-david) [reply src/components/manage/Blocks/Search/components/SelectStyling.jsx:36]: @dobri1408 this doesn't fix the fact that you can also have bottom text clipping when you use p or g
see the screenshot
4px-2-rows-demo

**Review by ichim-david — CHANGES_REQUESTED** [review]

**Comment by dobri1408** [comment]
I made sure that the padding is right now. I have put an 8 px padding on all sides.

**Comment by dobri1408** [comment]
I think the best solution is to increase the height of the container as the rows of the content are growing. Because the problem is now shifted to a text of 3 lines, instead of two.  I will try to approach this solution.

**Comment by ichim-david** [comment]
> I think the best solution is to increase the height of the container as the rows of the content are growing. Because the problem is now shifted to a text of 3 lines, instead of two. I will try to approach this solution.

I've seen that this fixes only 2 lines but you also have to think that it is a placeholder text, at some point, you have to keep it simple enough to be easily translatable and useful, I doubt that someone will use 3 lines of placeholder text.

**Review by sneridagh — APPROVED** [review]

**Pull request merged**